src/nnet3/nnet-normalize-component.h

```diff
@@ -224,6 +224,10 @@ class BatchNormComponent: public Component {
   const CuVector<BaseFloat> &Offset() const { return offset_; }
   const CuVector<BaseFloat> &Scale() const { return scale_; }
 
+  virtual const CuVector<double> &StatsSum() const { return stats_sum_; }
```

Comment:
Why do you make it a `virtual` method?

Reply:
there is no method of name `StatsSum ` in base calss, Should I use keyword override?

Reply:
No.

Then please remove `virtual`.

Reply:
@csukuangfj 
```
  CuVector<double> &StatsSum() { return stats_sum_; }
  CuVector<double> &StatsSumsq() { return stats_sumsq_; }
  double Count() { return count_; }
```
I have a problem here
```
TypeError: Unable to convert function return value to a Python type! The signature was
        (self: kaldi_pybind.nnet3.BatchNormComponent) -> kaldi::CuVector<double>
```

Reply:
Please paste the link to the code.

Reply:
Do not use double for any matrix types in Kaldi for pybind.

Reply:
but stats_sum_'s and stats_sumsq_ 's type are CuVector<double>
and offset_ scale_ also give me this bug
```
print(component.Offset().to_dlpack())
<capsule object "dltensor" at 0x7f28404c1780>

print(from_dlpack(component.Offset().to_dlpack()))
RuntimeError: CUDA error: invalid argument
```

Reply:
You do not need `stats_sum_` and `stats_sumsq_` in inference mode.

Reply:
Have you invoked `SetTestMode(true)` before 
`print(from_dlpack(component.Offset().to_dlpack()))` ?

Please re-check your code.

Reply:
the StatsMean and StatsVar in kaldi's model are equivalent to running_mean, running_var in pytorch, respectively.
but in kaldi's Read and Write function, there are some other operator, like 
```
Read
stats_sumsq_.AddVecVec(1.0, stats_sum_, stats_sum_, 1.0);
stats_sum_.Scale(count_);
stats_sumsq_.Scale(count_);
```
```
Write
CuVector<BaseFloat> mean(stats_sum_), var(stats_sumsq_);
  if (count_ != 0) {
    mean.Scale(1.0 / count_);
    var.Scale(1.0 / count_);
    var.AddVecVec(-1.0, mean, mean, 1.0);
  }
```
these operator are not necessary for pytorch's model, just use mean and var rather than offset and scale.